disk_objectstore/backup_utils.py

```diff
@@ -145,12 +154,28 @@ def call_rsync(  # pylint: disable=too-many-arguments
         :param dest_trailing_slash:
             Add a trailing slash to the destination path. This makes rsync interpret the
             destination as a folder and create it if it doesn't exists.
-
         """
 
-        assert "rsync" in self.exes
+        all_args = [
+            self.rsync_exe,
+            "-azh",
+            "--no-whole-file",
+        ]
+
+        capture_output = True
+        if LOGGER.isEnabledFor(logging.INFO):
+            capture_output = False
+            rsync_version = self.get_rsync_major_version()
```

Comment:
Would it make sense to do this operation once (in the constructor perhaps) when `rsync_exe` gets defined. It adds some overhead and is not really needed to call each time. Of course when people then update `BackupManager.rsync_exe` to a binary with a different version that would break, but I doubt that would happen. If you are afraid of this, you can make `rsync_exe` a "private" attribute `_rsync_exe`.

Reply:
good point, will do it in the constructor 👍